tests/test_commands.py

```diff
@@ -215,7 +238,7 @@ def start_requests(self):
         return []
 """
 
-    @contextmanager
+    @ contextmanager
```

Comment:
Accidental change?

Reply:
Oops, yea, sorry about that will revert back